style.css

```diff
@@ -16,279 +16,529 @@ body {
+  padding: 12px 16px;
+  color: #3965f2;
+  letter-spacing: 0.03em;
+  font-weight: 500;
+  font-size: 17px;
+  border-radius: 8px;
+}
+
+.card button:hover,
+.form-button:hover,
+#about .btn-resume:hover {
+  background-color: #3965f2;
+  color: #fff;
+  outline: none;
+}
+
+.card button:active,
+.form-button:active,
+#about .btn-resume:active {
+  color: #fff;
+  background-color: #2230d2;
+}
+
+.languages h1 {
+  font-weight: 400;
+  font-size: 20px;
+  color: #000;
+}
+
+.languages li {
+  list-style-type: none;
+}
+
+.languages .flex-box {
+  display: flex;
+  justify-content: space-between;
+  width: 90%;
+  margin: 10px 12px;
+}
+
+.languages .lang {
+  background-color: #f7f7f9;
+  margin: 36px 12px;
+  padding: 9px;
+  border-radius: 8px;
+  display: flex;
+  justify-content: start;
+  align-items: center;
+}
+
+.languages .lang p {
+  margin-left: 16px;
+}
+
+.fix {
+  margin: 0;
+  padding: 0;
+}
+
+hr {
+  border: 0.5px solid #dfe1e6;
+}
+
+.contact {
+  background-color: #6070ff;
+  padding-top: 0;
+  padding-bottom: 0;
+  border-radius: 100px 0 0 0;
+}
+
+.footer-container {
+  background: url(./images/contact-form-shape.svg) no-repeat 100% 30%;
+}
+
+.contact .text-container {
+  margin: 24px;
+  color: white;
+  text-align: center;
+}
+
+.contact .wrapper {
+  padding: 90px 0;
+  margin: 0 24px;
+}
+
+.contact .text-container h1 {
+  font-size: 40px;
+  font-weight: 700;
+}
+
+.contact .text-container p {
+  font-weight: 400;
+  color: #ebebff;
+}
+
+input,
+textarea {
+  background-color: #fff;
+  border: none;
+  width: 100%;
+  padding: 15px 16px;
+  border-radius: 8px;
+  margin: 15px 0;
+}
+
+.contact li {
+  list-style-type: none;
+  margin: 0;
+}
+
+input[value] {
+  color: #172b4d;
+  font-weight: 400;
+  font-size: 17px;
+}
+
+textarea::placeholder {
+  font-weight: 400;
+  font-size: 15px;
+  color: #b3bac5;
+}
+
+input:invalid {
+  /* box-shadow: 0 0 5px 1px red;   */
+}
+
+input:focus:invalid {
+  box-shadow: none;
+}
+
+/* for desktop */
+@media (min-width: 768px) {
+  .logo {
+    font-size: 18px;
+  }
+
   .nav-flex {
+    margin: 25px 155px;
+
+    /* height: 70px; */
+  }
+
+  nav button {
+    display: none;
+  }
+
+  nav ul {
     display: flex;
-    justify-content: space-between;
-    align-items: center;
-    margin: 15px 20px;
   }
 
-  nav .fa-sharp {
-    color: #6070ff;
-    font-size: 22px;
+  nav li {
+    list-style-type: none;
+    margin: 0 32px;
+  }
+
+  nav li a {
+    text-decoration: none;
+    font-size: 15px;
+    font-weight: 500;
+    color: #344563;
   }
 
   .headline {
-    background-image: url("./images/header-shapes\ mobile@2x.svg");
+    margin: 0 3px 0 34.6px;
+    background: url(./images/desktop/header-shapes\ mobile@2x.svg) no-repeat 100% 30%;
+    height: 100vh;
   }
 
   .headline .container {
-    margin: 0 20px;
-    height: 587.15px;
-    padding-top: 115px;
-    display: flex;
-    flex-direction: column;
+    margin: 0 155px;
+    padding: 70px 70px;
```

Comment:
- [ ] Good job so far 👏. As requested by the previous reviewer, please add the curving edge at the bottom left side of this headline section. Tools like [Clippy](https://bennettfeely.com/clippy/) can help you very much. Please try it out.

**your design**
![image](https://user-images.githubusercontent.com/90222110/193055793-0a00c477-7234-44d3-927d-b4176227c219.png)

**Figma design**
![image](https://user-images.githubusercontent.com/90222110/193056100-6c949551-c7d1-4495-ad88-d0030f098e89.png)